packages/ra-ui-materialui/src/input/RadioButtonGroupInput.tsx

```diff
@@ -95,6 +95,7 @@ export const RadioButtonGroupInput = (props: RadioButtonGroupInputProps) => {
         margin = 'dense',
         onBlur,
         onChange,
+        options,
```

Comment:
This is a code fix and not a Doc fix -> PR name should reflect that

Reply:
got it, I changed the PR title